content/blog/2020-06-29-optimizing-dvc-remotes.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Optimizing DVC Remotes
+date: 2020-06-29
```

Comment:
placeholder date

Reply:
I think this can be whatever you prefer.